pkg/encoding/fix.go

```diff
@@ -47,3 +53,18 @@ func fixPersistentVolumes(app *spec.App) error {
 	}
 	return nil
 }
+
```

Comment:
nit: can you add a comment here like the above 2 calls saying `// fix app.configMaps`